### ISSUE TITLE
KTO训练数据集报错

### ISSUE THREAD

**Issue by morettt** [modelscope/ms-swift#1513]
**Describe the bug**
我的kto格式的数据集是这样的
![image](https://github.com/user-attachments/assets/6578ad83-1484-4aa8-9655-9a1c1b38ce55)

这是我的执行命令：
#!/bin/bash
source activate swift

CUDA_VISIBLE_DEVICES=0 \
swift rlhf \
    --rlhf_type kto \
    --model_id_or_path  /root/autodl-tmp/Qwen2-7B-Chat \
    --model_type qwen2-7b-instruct \
    --beta 0.1 \
    --desirable_weight 1.0 \
    --undesirable_weight 1.0 \
    --sft_type  lora \
    --dataset /root/swift/examples/pytorch/llm/kto.json \
    --num_train_epochs  2  \
    --lora_target_modules  ALL  \
    --gradient_checkpointing  true  \
    --batch_size  1  \
    --learning_rate  5e-5  \
    --gradient_accumulation_steps  16  \
    --warmup_ratio  0.03  \
    --save_total_limit  2

但是最后还是一直报错KeyError: 'label'：
![image](https://github.com/user-attachments/assets/c8a534a4-29f4-4929-9031-0bd29ee9c2ca)

明明数据集内已经有label键 但是我明明就有啊？一直解决不了，搞了一整天了

**Comment by hjh0119** [comment]
sorry 已修复 https://github.com/modelscope/swift/pull/1515

**Comment by hjh0119** [comment]
label不要用字符串 用true, false

**Comment by morettt** [comment]
> 标签不要用字符串用true, false

可以了，感谢。我还想问问，kto数据集支持多轮对话训练吗？

**Comment by hjh0119** [comment]
支持的 [文档](https://github.com/modelscope/swift/blob/main/docs/source/LLM/%E8%87%AA%E5%AE%9A%E4%B9%89%E4%B8%8E%E6%8B%93%E5%B1%95.md)中有KTO的自定义数据集格式样例
